exoctk/exoctk_app/app_exoctk.py

```diff
@@ -208,6 +205,12 @@ def limb_darkening():
 
             profile_tables.append(html_table)
 
+            # Add the profile to the form inputs
+            form_input[profile] = 'true'
```

Comment:
Is there a reason to use the string `'true'` here instead of the python boolean `True`?